tensorboard/plugins/profile/pod_viewer/details_card/BUILD

```diff
@@ -0,0 +1,19 @@
+package(default_visibility = ["//tensorboard:internal"])
+
+load("//tensorboard/defs:web.bzl", "tf_web_library")
+
+licenses(["notice"])  # Apache 2.0
+
+tf_web_library(
+    name = "details_card",
+    srcs = [
+        "details-card.html",
+        "details-card.ts",
+    ],
+    path = "/pod-viewer",
```

Comment:
Acknowledged.
I would prefer /pod-viewer/details-card, so that each component is put into its own folder. It has some benefits when compiling the TS back to JS file to move into google3.
I would keep this path for now to be consistent with other tools under profile plugin.